packages/boot/src/components/app/index.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { createRoot } from '@wordpress/element';
+import { dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import Router from './router';
+import { STORE_NAME } from '../../store';
+import type { MenuItem } from '../../store/types';
+
+function App() {
+	return <Router />;
+}
+
+export async function init( menuItems: MenuItem[] ) {
+	// Import the store to ensure it's registered
+	await import( '../../store' );
```

Comment:
Curious about this. Wouldn't the above `import { STORE_NAME } from '../../store';` ensure that?